### ISSUE TITLE
-f|--framework is listed in docs for "dotnet new console" but is hidden in CLI help

### ISSUE THREAD

**Issue by shinfd** [dotnet/docs#17809]
Cannot tell which one is supposed to be authoritative, tying the report to docs.

Snippet of the output from `dotnet new console --help`:

```
Console Application (C#)
Author: Microsoft
Description: A project for creating a command-line application that can run on .NET Core on Windows, Linux and macOS
Options:
  --langVersion  Sets the LangVersion property in the created project file
                 text - Optional

  --no-restore   If specified, skips the automatic restore of the project on create.
                 bool - Optional
                 Default: false / (*) true
```

```
C:\&gt;dotnet --info
.NET Core SDK (reflecting any global.json):
 Version:   3.1.201
 Commit:    b1768b4ae7

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.18363
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\3.1.201\

Host (useful for support):
  Version: 3.1.3
  Commit:  4a9f85e9f8

.NET Core SDKs installed:
  3.1.201 [C:\Program Files\dotnet\sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.App 3.1.3 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 3.1.3 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 3.1.3 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

To install additional .NET Core runtimes or SDKs:
  https://aka.ms/dotnet-download
```

Regards.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0b952f8a-18e1-4843-0741-6ca877fb303a
* Version Independent ID: d80dfb96-bc86-a960-abec-255dd907d04d
* Content: [dotnet new command - .NET Core CLI](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-new#feedback)
* Content Source: [docs/core/tools/dotnet-new.md](https://github.com/dotnet/docs/blob/master/docs/core/tools/dotnet-new.md)
* Product: **dotnet-core**
* Technology: **dotnet-cli**
* GitHub Login: @tdykstra
* Microsoft Alias: **tdykstra**

**Comment by shinfd** [comment]
Ah. This may be due to the fact I have got template for only single version installed? My installation is a fresh install of .NET Core SDK via VS Community 2019.

Found a [comment](https://github.com/dotnet/docs/issues/5961#issuecomment-397468620) by seancpeters in #5961:
> When showing help on a template, e.g. dotnet new web -h, if only 1 framework is available, it's not listed in the help options because it cannot be varied. But if multiple frameworks are available for a template, the help display with show them all.

Might be appropriate to close this one then, if above issue will address the discrepancy between doc and CLI help, depending on the installation.
